numpy/core/tests/test_multiarray.py

```diff
@@ -4426,6 +4426,12 @@ def test_nan(self):
         expected = np.array([-1., np.nan, 0.5, 1., 0.25, np.nan])
         assert_array_equal(result, expected)
 
+    def test_complex(self):
+        val = np.array([0+7j, 1+6j, 2+5j, 3+4j])
+        result = val.clip(1+5j, 2+6j)
+        expected = np.array([1+6j, 1+6j, 2+5j, 2+5j])
```

Comment:
According to the clip definition, values outside the interval are clipped to the interval edges, so the value of expected should be np.array([1+5j, 1+6j, 2+5j, 2+6j]) ?